protocol/messenger_test.go

```diff
@@ -874,6 +874,67 @@ func (s *MessengerSuite) TestRetrieveTheirPrivateGroupChat() {
 	s.Require().NotNil(actualChat.LastMessage)
 }
 
+// Test receiving a message on an existing private group chat
+func (s *MessengerSuite) TestChangeNameGroupChat() {
+	var response *MessengerResponse
+	theirMessenger := s.newMessenger(s.shh)
+	response, err := s.m.CreateGroupChatWithMembers(context.Background(), "id", []string{})
+	s.NoError(err)
+	s.Require().Len(response.Chats, 1)
+
+	ourChat := response.Chats[0]
+
+	err = s.m.SaveChat(ourChat)
+	s.NoError(err)
+
+	members := []string{"0x" + hex.EncodeToString(crypto.FromECDSAPub(&theirMessenger.identity.PublicKey))}
+	_, err = s.m.AddMembersToGroupChat(context.Background(), ourChat.ID, members)
+	s.NoError(err)
+
+	// Retrieve their messages so that the chat is created
+	err = tt.RetryWithBackOff(func() error {
+		var err error
+		response, err = theirMessenger.RetrieveAll()
+		if err == nil && len(response.Chats) == 0 {
+			err = errors.New("chat invitation not received")
+		}
+		return err
+	})
+	s.Require().NoError(err)
+
+	_, err = theirMessenger.ConfirmJoiningGroup(context.Background(), ourChat.ID)
+	s.NoError(err)
+
+	err = tt.RetryWithBackOff(func() error {
```

Comment:
Add a comment as above like `// Retrieve messages to verify group was joined`. Maybe this could be a helper method called `retrieveOrError()` or something like that. Since we do it three times.